plugins/pcl/io/PCLVisualizer.cpp

```diff
@@ -140,16 +140,16 @@ CREATE_SHARED_PLUGIN(1, 0, PclVisualizer, Writer, s_info)
 
 std::string PclVisualizer::getName() const { return s_info.name; }
 
-void PclVisualizer::write(const PointBuffer& data)
+void PclVisualizer::write(const PointViewPtr view)
 {
     // Determine XYZ bounds
-    BOX3D const& buffer_bounds = data.calculateBounds();
+    BOX3D const& buffer_bounds = view->calculateBounds();
 
     typedef XYZIRGBA PointType;
 
-    // Convert PointBuffer to a PCL PointCloud
+    // Convert PointView to a PCL PointCloud
     pcl::PointCloud<PointType>::Ptr cloud(new pcl::PointCloud<PointType>);
-    pclsupport::PDALtoPCD(const_cast<PointBuffer&>(data), *cloud, buffer_bounds);
+    pclsupport::PDALtoPCD(const_cast<PointViewPtr>(view), *cloud, buffer_bounds);
```

Comment:
`Error	460	error C2440: 'const_cast' : cannot convert from 'const pdal::PointViewPtr' to 'pdal::PointViewPtr'`